releases/24.0.0.1/full/Dockerfile.ubi.ibmjava8

```diff
@@ -3,7 +3,7 @@ FROM ibmjava:8-ubi AS getRuntime
 USER root
 
 ARG LIBERTY_VERSION=24.0.0.1
-ARG LIBERTY_SHA={replace_with_correct_sha}
+ARG LIBERTY_SHA=98c9adcf842c536fc9862959b4832fe099b540209eec3c9c8987769800d96df4
```

Comment:
The SHA validation below checks for the sha1 value. This looks like the sha256 value.